crates/astria-sequencer/src/app.rs

```diff
@@ -1167,6 +1170,58 @@ mod test {
         assert_eq!(validator_updates.get(&pub_key.into()).unwrap(), &update);
     }
 
+    #[tokio::test]
+    async fn app_deliver_tx_ibc_relayer_change_addition() {
```

Comment:
Should we add a test to validate it doesn't change when using the wrong sudo key?